src/test/kotlin/com/exactpro/th2/codec/ProcessorTest.kt

```diff
@@ -178,40 +332,45 @@ class ProcessorTest {
         Assertions.assertTrue(result.getGroups(0).messagesList[0].hasMessage())
         result.getGroups(0).messagesList[0].message.let {
             Assertions.assertEquals(ERROR_TYPE_MESSAGE, it.messageType)
-            Assertions.assertEquals(originalProtocol, it.metadata.protocol)
+            Assertions.assertEquals(ORIGINAL_PROTOCOL, it.metadata.protocol)
+            Assertions.assertTrue(it.hasField(ERROR_EVENT_ID))
+            Assertions.assertTrue(it.hasField(ERROR_CONTENT_FIELD))
         }
 
         Assertions.assertTrue(result.getGroups(0).messagesList[1].hasRawMessage())
         result.getGroups(0).messagesList[1].rawMessage.let {
-            Assertions.assertEquals(wrongProtocol, it.metadata.protocol)
+            Assertions.assertEquals(WRONG_PROTOCOL, it.metadata.protocol)
+            Assertions.assertTrue(it.descriptorForType.findFieldByName(ERROR_EVENT_ID) == null)
+            Assertions.assertTrue(it.descriptorForType.findFieldByName(ERROR_CONTENT_FIELD) == null)
```

Comment:
> descriptorForType.findFieldByName

Why not use `hasField` instead?

Reply:
Function `fun Message.hasField(key: String) : Boolean` is defined in `com.exactpro.th2.common.message.MessageUtils.kt`, but there's no a similar method for `RawMessage`.

Reply:
Didn't notice that it's a raw message. Raw message doesn't have fields so this check doesn't make sense